py_ocsf_models/objects/resource_details.py

```diff
@@ -32,7 +32,7 @@ class ResourceDetails(BaseModel):
     criticality: Optional[str]
     data: Optional[dict[str, object]]
     group: Optional[Group]
-    labels: Optional[list[str]]
+    labels: Optional[dict]
```

Comment:
This will not comply with the OCSF specification https://schema.ocsf.io/1.2.0/objects/resource_details?extensions=. `labels` must be a String Array.